java-security/src/main/java/com/sap/cloud/security/servlet/XsuaaTokenAuthenticator.java

```diff
@@ -103,6 +103,7 @@ public TokenAuthenticationResult validateRequest(ServletRequest request, Servlet
 		if (request instanceof HttpServletRequest && response instanceof HttpServletResponse) {
 			HttpServletRequest httpRequest = (HttpServletRequest) request;
 			String authorizationHeader = httpRequest.getHeader(HttpHeaders.AUTHORIZATION);
+			LOGGER.debug("Authorization header: {}", authorizationHeader);
```

Comment:
unfortunately we can't add the encoded jwt into the log, someone could make use of it (until it gets expired).... If its helpful we could print the first 20 characters sth like "bearer abcdefg123456..."